src/client.js

```diff
@@ -10,14 +10,14 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 (function() {
-"use strict";    
```

Comment:
Lots of changes in blank spaces is making the diff confusing. If you want to do some cleanup, can you do a separate PR?
In fact, it would be good to have a separate PR which removes whitespaces and add linting rules to make sure this remains correct